portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,75 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.ArrayList;
+import java.util.List;
 import java.io.IOException;
```

Comment:
`import java.io.IOException;` should go before `import java.util...` alphabetically